tests/data/pickings/missing_samples.json

```diff
@@ -0,0 +1,147 @@
+{
+    "data": [
+        {
+            "id": "2",
+            "type": "plates",
+            "links": {
+                "self": "http://127.0.0.1:3000/api/v2/plates/2"
```

Comment:
should we be hardcoding URLs like this?

Reply:
Thanks. I added complete response data for testing to show how they look like. The URLs are not used by the tests. They could be changed into "example.com" URLs . Does it worth changing them now?